nixos/modules/programs/mininet.nix

```diff
@@ -0,0 +1,40 @@
+# Global configuration for mininet
+# kernel must have NETNS/VETH/SCHED
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg  = config.programs.mininet;
+
+  # pyEnv = pkgs.python.withPackages(ps: [ ps.mininet-python ]);
+
+  generatedPath = with pkgs; makeSearchPath "bin"  [
+    # mn will stop if no telnet is available
+    # telnet
+    iperf ethtool iproute socat
```

Comment:
There is no telnet in the list here. All mnexec should be rather wrapped in the package itself.

Reply:
telnet is called by the python mn binary